packages/react/src/SelectPanel/SelectPanel.tsx

```diff
@@ -132,10 +132,11 @@ interface SelectPanelBaseProps {
     variant: 'info' | 'warning' | 'error'
   }
   onCancel?: () => void
+  variant?: 'anchored' | 'modal'
 }
 
 export type SelectPanelProps = SelectPanelBaseProps &
-  Omit<FilteredActionListProps, 'selectionVariant'> &
+  Omit<FilteredActionListProps, 'selectionVariant' | 'variant'> &
```

Comment:
Note for reviewer: There used to be a `variant` prop in FilteredActionList but that's not used at all with SelectPanel. [react query with no results ↗](https://primer-query.githubapp.com/?rattribute=variant&reattribute=variant&repattribute=variant&repoattribute=variant&name=selectpanel&repo=github%2Fgithub&attribute=variant&entrypoint=%22%40primer%2Freact%22)